src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.containers') {
+    .clr-form-control {
+        margin-top: $clr-forms-baseline * 4;
+        display: flex;
+        flex-direction: column;
+
+        .clr-control-label {
+            display: block;
+            color: $clr-forms-label-color;
+            font-size: calc(#{$clr-forms-baseline} * 2 + 1px);
+            font-weight: 600;
+            line-height: $clr-forms-baseline * 3;
+        }
+
+        .clr-control-container {
+            display: inline-block;
+
+            .clr-subtext {
```

Comment:
`.clr-form-control .clr-control-container .clr-subtext` et al.

...could be better as...

`.clr-control-container .clr-subtext` or `.clr-form-control .clr-subtext` et al.

No need to use css selectors for DOM mapping.